expression/builtin_miscellaneous_vec.go

```diff
@@ -15,6 +15,7 @@ package expression
 
 import (
 	"encoding/binary"
+	"github.com/google/uuid"
```

Comment:
Please group this line into github.com/...

Reply:
ok, i will refine it